spec/runner_spec.rb

```diff
@@ -0,0 +1,185 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'validate_environment_varables!' do
+  let(:envvars) { Jacaranda::Runner.required_environment_variables }
+  subject { -> { Jacaranda::Runner.validate_environment_variables! } }
+
+  context 'when all environment variables are set' do
+    before(:each) { envvars.each { |var| set_environment_variable(var, Faker::Name.first_name) } }
+
+    it 'does not exit' do
+      is_expected.to_not raise_error
+    end
+  end
+
+  context 'when any environment variables are not set' do
+    before(:each) { envvars.each { |var| unset_environment_variable(var) } }
+
+    it 'exits' do
+      is_expected.to raise_error(SystemExit)
+    end
+  end
+
+  after(:each) { restore_env }
+end
+
+describe 'Jacaranda#run' do
```

Comment:
This description is confusing because none of the tests actually test the "Jacaranda#run" method. Also "#" is for instance methods, so it should be `describe "Jacaranda.run" do`.